config/kibana.yml

```diff
@@ -40,8 +40,8 @@
 # the username and password that the Kibana server uses to perform maintenance on the Kibana
 # index at startup. Your Kibana users still need to authenticate with Elasticsearch, which
 # is proxied through the Kibana server.
-#elasticsearch.username: "kibana_system"
-#elasticsearch.password: "pass"
+elasticsearch.username: "kibana_system"
```

Comment:
You must remove these edits before merging!

Reply:
This is why we added a codeowner on this file actually 😄 . Once reverted, you can remove the review request from the platform team.

Reply:
Thanks @pgayvallet - I have reverted the change and removed review request. Sorry about that.